ports/mimxrt/boards/MIMXRT1010_EVK/mpconfigboard.h

```diff
@@ -3,6 +3,7 @@
 
 #define BOARD_FLASH_SIZE (16 * 1024 * 1024)
 
-#define MICROPY_HW_LED_PINMUX  IOMUXC_GPIO_11_GPIOMUX_IO11
-#define MICROPY_HW_LED_PORT    GPIO1
-#define MICROPY_HW_LED_PIN     11
+// i.MX RT1010 EVK has 1 board LED
+#define MICROPY_HW_LED_PIN (GPIO_11)
```

Comment:
perhaps this should be `MICROPY_HW_LED1_PIN` to support multiple LEDs in the future?